config/examples/vmagent_rbac.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: vmagent
-  namespace: monitoring-system
+  namespace: default
```

Comment:
lets leave it as is.
 Lets leave it as is.

 Namespace can be changed with kustomize or with sed replacement, as mentioned at docs.

 For kubectl apply manual changing is needed any way.